### PR TITLE
211 remove bootstrap (connects #211)

### PR DESCRIPTION
**Pull request by razu9861**
I found bootstrap class used in pagenotfound component.

### CODE REVIEW

File: src/app/home/navigation.component.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ import { Router } from '@angular/router';
     <ul>
       <li *ngFor="let comp of components"><a [routerLink]="'/' + comp.link">{{comp.name.toUpperCase()}}</a></li>
       <li><a href="#" class="km-logout" (click)="logoutClick()">LOGOUT</a></li>
-      <li><a routerLink="/manager"><i class="fa fa-user" aria-hidden="true"></i></a></li>
+      <li><a routerLink="/manager"><mat-icon>person</mat-icon></a></li>
```
Comment (Pr0chin) [inline src/app/home/navigation.component.ts]: Nice but can u resize icon as it has effected in navigation-bar or else there might be material  icon like medium..

Comment (razu9861) [reply src/app/home/navigation.component.ts]: this PR only deals with removing bootstrap, popper and fontawesome and repllacing fa icon with material icon not ablout styling or designing

Comment (Pr0chin) [reply src/app/home/navigation.component.ts]: then why did u change code right here navigation.component.ts as it results :
see

Comment (razu9861) [reply src/app/home/navigation.component.ts]: as far as i was told to keep everything as it is and not to touch designing par or styling part

**Pull request merged**